HttpTriggerSqlClient/__init__.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import urllib
+from datetime import datetime
+import uuid
+import os
+
+import azure.functions as func
+from sqlalchemy import create_engine, Table, DateTime, Column, MetaData, String
+
+engine = None
+
+# Information used to build the connection string,
+# stored in the local.settings.json file.
+server = os.environ["DATABASE_SERVER"]
```

Comment:
One improvement here is to use `os.getenv` instead of `os.environ`. `os.environ` will throw an KeyNotFound exception if "DATABASE_SERVER" environment variable is not found, while `os.getenv` returns None if the value is not provided.

Reply:
@Hazhzeng should we also update the guidance here? https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=azurecli-linux%2Capplication-level#environment-variables

Reply:
Yes, this is a good idea!